source/howto/tourism/alpinebits.rst

```diff
@@ -0,0 +1,121 @@
+.. _ab-howto:
+
+How to access AlpineBits data
+=============================
+
+The AlpineBits Alliance strives to develop and to spread a standard
+format to exchange data. In this howto, we show how to retrieve data
+from the AlpineBits endpoint located at
+https://alpinebits.opendatahub.bz.it/AlpineBits, by using the (Linux)
+command line--in particular the :command:`curl` application, and the
+popular `Postman` API development environment.  An example call can be
+seen :ref:`at the bottom <ab-request>` of this section.
+
+.. note:: Alternative command line programs like :command:`wget` can
+   be used as well: simply adapt the parameters described in the
+   remainder of this section.
+
+The first important thing to mention is that requests to this endpoint
+need a :strong:`POST` method and an authentication token, therefore
+you should specify options :literal:`--request POST` and a header
+requiring `Basic` authorization and containing your token:
+:literal:`--header 'Authorization: Basic <your-token>'`
```

Comment:
For the Open Data Hub there is no authentication needed, so you can basically insert anything such as "anonymous" here. Although we strongly suggest to insert your mailaddress or name for debugging reasons.

Reply:
I also added a note with this information